grafana/kvdb/README.md

```diff
@@ -0,0 +1,7 @@
+# KVDB Grafana dashboards
+
+This directory stores the Grafana dashboard JSON templates for the following introspector KVDB metrics:
```

Comment:
Why not have all of them in a single dashboard and remove the folder ?

Reply:
We should probably keep all the individual ones, and in stead add a combined one as well for optionality. Will add this later today.

Reply:
I'm not sure it is worth exporting individual panels, or if it can actually work for the panels as currently defined. I also tried to import the PR JSONS and it doesn't work.

For things to work on import we also have to export the variables which are included in the `templating` JSON node.

Reply:
That helps! In that case, we should try with the entire JSON model. Will add it now (+ descriptions).